### PR TITLE
Add the navigation bar with the topics

### PR DESCRIPTION
**Pull request by mariamera**
Closes [Ticket](https://trello.com)

### CODE REVIEW

**Comment by junior1q94** [comment]
LGTM
FYI: @raulgomezc  @oscarpolanco

**Review by junior1q94 — APPROVED** [review]

**Comment by EnmanuelParache** [comment]
I think the content section should scroll along with the page so that I don't have to scroll back up to change to a section. @mariamera

**Review by oscarpolanco — APPROVED** [review]
LGTM

File: docs/conf.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
+
```
Comment (raulgomezc) [inline docs/conf.py]: Remove surplus line

**Review by raulgomezc — CHANGES_REQUESTED** [review]

**Review by raulgomezc — APPROVED** [review]
LGTM

**Pull request merged**